package.json

```diff
@@ -17,10 +17,19 @@
   },
   "homepage": "https://github.com/cf-lms/lms#readme",
   "devDependencies": {
+    "babel-core": "^6.3.17",
+    "babel-loader": "^6.2.0",
+    "babel-preset-react": "^6.3.13",
     "chai": "^3.4.1",
-    "mocha": "^2.3.4"
+    "gulp-babel": "^6.1.1",
+    "marked": "^0.3.5",
+    "mocha": "^2.3.4",
+    "webpack-stream": "^3.1.0"
   },
   "dependencies": {
-    "express": "^4.13.3"
```

Comment:
these are not accidental changed. They are to keep the dependencies in alphabetical order.
 Add a line note